packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -614,6 +596,10 @@ const DatePickerInput = (props) => {
     } else if (inputType === DateUtil.inputType.DAY) {
       handleDayInputKeydown(event);
     }
+
+    if (DateUtil.isMac() && !event.key.match(/^[0-9]/g) && !(event.keyCode === KEY_BACK_SPACE || event.keyCode === KEY_DELETE || event.keyCode === KEY_TAB || event.keyCode === KEY_RIGHT || event.keyCode === KEY_LEFT)) {
```

Comment:
why mac specific here ? Did we miss adding here as well: https://github.com/cerner/terra-framework/pull/2017/files#diff-37c08a2b0d74d4e14dd7fc4e7e160bb6ac190dd983aebaaf30397c71621b735dR479